code/game/objects/structures/morgue.dm

```diff
@@ -153,7 +153,7 @@ GLOBAL_LIST_EMPTY(bodycontainers) //Let them act as spawnpoints for revenants an
 
 /obj/structure/bodycontainer/morgue/examine(mob/user)
 	..()
-	to_chat(user, "<span class='notice'>Alt-click [src] to [beeper ? "disable" : "enable"] the alarm.</span>")
+	to_chat(user, "<span class='notice'>The speaker is [beeper ? "enabled" : "disabled"]. Alt-click to toggle it.</span>")

```

Comment:
change ```notice``` to ```notice bold```

Reply:
why don't you just bold the enabled/disabled if you're going to do that.

Reply:
It's not important enough to deserve it